src/tiled/createobjecttool.h

```diff
@@ -66,6 +69,8 @@ class CreateObjectTool : public AbstractObjectTool
     ObjectGroupItem *mObjectGroupItem;
     MapObjectItem *mNewMapObjectItem;
     MapObjectItem *mOverlayPolygonItem;
+    bool mExtending;
+    bool mExtendingFirst;
```

Comment:
I removed `CreateObjectTool::` but added `CreateMultipointObjectTool::`, because without any `className::`, their was some runtime error (the window wasn't responding at all). I wasn't able to understand why this happened.